mappings/net/minecraft/client/render/entity/model/TurtleEntityModel.mapping

```diff
@@ -1 +1,2 @@
 CLASS net/minecraft/class_614 net/minecraft/client/render/entity/model/TurtleEntityModel
+	FIELD field_3594 shell Lnet/minecraft/class_630;
```

Comment:
Not the shell, it's a very little part of the model. See images below (first is with this part visible, second not visible)
![2020-09-02_17 42 22](https://user-images.githubusercontent.com/16228338/92005790-302c7380-ed44-11ea-8823-3802527c6536.png)
![2020-09-02_17 42 32](https://user-images.githubusercontent.com/16228338/92005801-328ecd80-ed44-11ea-83dc-f5cfff64b231.png)

Reply:
is "belly" better? That's what I went with in the latest commit.

Reply:
Yeah definitely better

Reply:
'Plastron' if you wanted to be [anatomically correct](https://github.com/FabricMC/yarn/blob/51380a15d46ab03ac413b713861d9580c66c580b/mappings/net/minecraft/client/render/entity/model/ChickenEntityModel.mapping#L4)